bedrock-devnet/devnet/__init__.py

```diff
@@ -142,10 +142,12 @@ def devnet_l1_allocs(paths):
     init_devnet_l1_deploy_config(paths)
 
     fqn = 'scripts/Deploy.s.sol:Deploy'
-    # Use foundry pre-funded account #1 for the deployer
     run_command([
-        'forge', 'script', '--chain-id', '900', fqn, "--sig", "runWithStateDump()", "--private-key", "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
-    ], env={}, cwd=paths.contracts_bedrock_dir)
```

Comment:
I'm not too familiar with the devnet setup—who is the new deployer now?

Reply:
The default foundry sender is used